common/src/main/java/tachyon/exception/ExceptionMessage.java

```diff
@@ -40,6 +40,11 @@
   TIER_ALIAS_NOT_FOUND("Tier with alias {0} not found"),
   TIER_VIEW_ALIAS_NOT_FOUND("Tier view with alias {0} not found"),
 
+  // storageDir
+  ADD_EXISTING_BLOCK("blockId {0} exists"),
```

Comment:
for these three messages, maybe also print out which storage dir we operate on when throwing out these errors. e.g., `blockId {0} exists in {1}` and etc?